src/app/Job-Service/job.service.ts

```diff
@@ -14,29 +14,29 @@ export class JobService {
   constructor(private httpServ: HttpClient) { }
 
   saveJob(job: Job) {
-    return this.httpServ.post<Job>(this.url + '/saveJob', job);
+    return this.httpServ.post<Job>(this.url, job);
   }
 
   updateJob(job: Job) {
-    return this.httpServ.patch<boolean>(this.url + '/updateJob', job);
+    return this.httpServ.patch<boolean>(this.url, job);
```

Comment:
So this method will always return a 405 Method Not Allowed, since the JobController on the backend does not have a PatchMapping annotation. In fact, the backend does not currently support any update Job functionality. This method appears to have already existed previously, but since it isn't supported at the moment, this method should be removed entirely. Perhaps in a future sprint, that functionality could be added back in if needed.